dom.bs

```diff
@@ -684,6 +700,13 @@ The <dfn attribute for=Event><code>bubbles</code></dfn> and
 <dfn attribute for=Event><code>cancelable</code></dfn> attributes
 must return the values they were initialized to.
 
+The <dfn attribute for=Event><code>returnValue</code></dfn>
+attribute must be initialized to true when an <a>event</a> is created.
```

Comment:
I don't think this makes sense. You want to instead describe this as a getter and a setter. The default value is already evident from the default value of the canceled flag after all.

See https://url.spec.whatwg.org/#dom-url-protocol for an example of defining a getter and a setter.